sphinx/theming.py

```diff
@@ -22,7 +22,7 @@
 if sys.version_info >= (3, 10):
     from importlib.metadata import entry_points
 else:
-    from importlib_metadata import entry_points
+    from importlib_metadata import entry_points  # type: ignore[import-not-found]
```

Comment:
again, this shouldn't be required

Reply:
Well, if I didn't put it, mypy complains =/

Reply:
that makes me think the import guard is wrong

sounds like `if sys.version_info >= (3, 10):` is the wrong python version to guard against